packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit_webview_widget.dart

```diff
@@ -619,7 +620,13 @@ class WebViewWidgetProxy {
   }
 
   /// Constructs a [WKNavigationDelegate].
-  WKNavigationDelegate createNavigationDelegate() {
-    return WKNavigationDelegate();
+  WKNavigationDelegate createNavigationDelegate({
+    void Function(
+      WKWebView webView,
+      String? url,
+    )?
```

Comment:
Do you want to typedef this somewhere? It's shown up a few times.

Reply:
I assumed we wanted to follow https://dart.dev/guides/language/effective-dart/design#prefer-inline-function-types-over-typedefs. The most important line being:

> But in most cases, users want to see what the function type actually is right where it’s used, and the function type syntax gives them that clarity.

Reply:
Ah, I didn't realize there was a style guideline for that. SG.

(Does this read better or worse without the comma after `url`?)

Reply:
I noticed that too and already removed the comma!